app/views/admin/events/show.html.haml

```diff
@@ -69,9 +69,10 @@
               %span{ 'class' => [icon_for_todo(progress_status['registered']), 'fa-lg'] }
           %tr
             %td
-              - if @event.speakers.count == 1
+              - speakers_count = @event.speakers.count
+              - if speakers_count == 1
                 = link_to 'Fill out speaker\'s biography', edit_admin_user_path(@event.speakers.first)
-              - else
+              - elsif speakers_count > 1
```

Comment:
You can use the [case statement](https://www.rubyguides.com/2015/10/ruby-case/) for this. :wink:

Reply:
I will need some help here. I am not sure about fixing the upper-limit for number of speakers (say, `when 1..1000` or `when 1..2000`). Else will have to use some function to compare. Which will be the better option?

Reply:
There is a new endless range in Ruby 2.6 that could be used here: https://anamaria.martinezgomez.name/2018/12/21/ruby2_6.html

But we are using Ruby 2.5 :sob: 

I find it good as it is now. :+1: :+1: